default/scripting/species/SP_ANCIENT_GUARDIANS.focs.txt

```diff
@@ -0,0 +1,90 @@
+Species
+    name = "SP_ANCIENT_GUARDIANS"
+    description = "SP_ANCIENT_GUARDIANS_DESC"
+    gameplay_description = "SP_ANCIENT_GUARDIANS_GAMEPLAY_DESC"
+
+    tags = [ "ROBOTIC" "NO_INDUSTRY" "NO_RESEARCH" "AVERAGE_SUPPLY" "DESTROYED_ON_CONQUEST"]
+
+    foci = [
+        [[HAS_INDUSTRY_FOCUS]]
+        [[HAS_RESEARCH_FOCUS]]
+        [[HAS_GROWTH_FOCUS]]
+        [[HAS_ADVANCED_FOCI]]
```

Comment:
Morlic had an [unanswered question](https://github.com/freeorion/freeorion/pull/837#discussion_r81445479) here regarding foci.
At least one is likely required, a populated planet with no available focus could lead to issues.

Any consideration for using `FOCUS_PROTECTION`, and setting it as the preferred focus?
(removing all other foci, and adjusting values as you see fit)